index.html

```diff
@@ -0,0 +1,131 @@
+        <div onclick="menuPopup()" id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div> 
+        <div onclick="menuPopup()" id="closeButton"> &times; </div>
+      </nav>
+    </header>
+
+    <section class="mobilemain">
+        <article class="home-page" id="#home" title="home-page">
+          <h3 class="intro-title-one"><q>cakey-cakey by Marianna</q></h3>
+          <h1 class="intro-title-two">CAKE-LOVERS WORKSHOP FOR MINORITY GROUP 2023</h1>
+            <p class="explanation-paragraph" alt="explanation-paragraph">
+              A workshop that will allows low income individuals to
+              exhibit with low income to get source of income why can sell their cake to the host community and make money out of it</p>
+          <h4 class="event-date" alt="event-date"> 2023.05.25(THU) ~ 26(FRI)</h4>
+          <div class="copyright">&copy; Nairobi, Kenya </div>
+        </article>
+
+        <article id="packages-offered" title="packages-offered">
+            <h5 class="packages-title" id="packages"> Packages Offered</h5>
+
+            <section class="card-stack">
+              <div class="cards card1" alt="Card1">
+                <img class="cards-img card1-img" src="images/program_icon_01.png" alt="Card1-image">
+                  <h5 class="card-title">lecture</h5>
+                  <p class="card-text">Lorem ipsum dolor sit amet conseinus cabo, perferendis veniam neque!.</p>
+              </div>
+
+              <div class="cards card2" alt="Card2">
+                <img class="cards-img card2-img" src="images/program_icon_02.png" alt="Card2-image">
+                  <h5 class="card-title">lecture</h5>
+                  <p class="card-text">Lorem ipsum dolor sit amet conseinus cabo, perferendis veniam neque!</p>
+              </div>
+
+              <div class="cards card3" alt="Card3">
+                <img class="cards-img card3-img" src="images/program_icon_03.png" alt="Card3-image">
+                  <h5 class="card-title">lecture</h5>
+                  <p class="card-text">Lorem ipsum dolor sit amet conseinus cabo, perferendis veniam neque!.</p>
+              </div>
+
+              <div class="cards card4" alt="Card4;">
+                <img class="cards-img card4-img" src="images/program_icon_04.png" alt="Card4-image">
+                  <h5 class="card-title">lecture</h5>
+                  <p class="card-text">Lorem ipsum dolor sit amet conseinus cabo, perferendis veniam neque!ard</p>
+              </div>
+            </section>
+
+            <a href="#" class="view-all">VIEW ALL</a>
+            <button class="join-us">Join Us Today!</button>
+        </article>
+        
+        <section id="speakers" title="speakers">
+          <h5 class="speakers-maintitle">Featured Speakers</h5>         
+            <div id="speaker-cards"> 
+              <!-- Renders all speakers dynamically -->      
+            </div> 
```

Comment:
- [ ] You did great implementing the dynamic rendering of the speakers, :+1: You also did great implementing the see more button for the mobile screen.
- - As part of the requirement for this Capstone is to replicate the given design as much as you can, the provided design has only 2 of the speaker visible in the mobile screens, but that's not the case in your design, I'll suggest you also implement this so that your project looks like the given design. :wave: :smile: 

- - Your Design:
![Screenshot from 2023-05-24 14-42-09](https://github.com/MariamOsman11/CAPSTONE-PROJECT-ONE/assets/107632780/7bef4179-e461-48ad-bad8-13350003ccc6)

- - Behance Design:
![Screenshot from 2023-05-24 14-42-25](https://github.com/MariamOsman11/CAPSTONE-PROJECT-ONE/assets/107632780/dc8d9642-5a2d-4a24-b151-01b77c44613c)

- - You can achieve this in your project by adding this to your CSS before the media query `#speaker-cards .speaker-cards:nth-child(n+3){
  display: none;
} ` and then add this in the media query: `  #speaker-cards .speaker-cards:nth-child(n+3){
    display: block;
  }`

Reply:
Not yet implemented. 🙋‍♂️